.github/workflows/mad_regenerate-models.yml

```diff
@@ -0,0 +1,56 @@
+name: Regenerate framework models
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 2 * * *"
+
+jobs:
+  regenerate-models:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        slug: ["placeholder"]
+        ref: ["placeholder"]
+        include:
+          - slug: "apache/commons-io"
+            ref: "8985de8fe74f6622a419b37a6eed0dbc484dc128"
+        exclude:
+          - slug: "placeholder"
+            ref: "placeholder"
```

Comment:
Why do we need the `placeholder`s here (, which are excluded)?

Reply:
In order to create a paramterized job without the combinatoric nature of a matrix build (and all axis are required to have at least one value)